torch/_functorch/config.py

```diff
@@ -32,11 +32,32 @@
 cse = True
 
 # Restricts the amount of computation AOTAutograd can do.
-max_dist_from_bw = 3
+max_dist_from_bw = 1000
```

Comment:
where is this number coming from ? if it's this large, isnt even worth having ?

Reply:
Yeah, it's basically removed now - keeping it for now as a knob in case somebody finds it necessary in the short term.